WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.m

```diff
@@ -47,6 +47,17 @@
 static NSTimeInterval NotificationsUndoTimeout          = 4;
 static NSString const *NotificationsNetworkStatusKey    = @"network_status";
 
+static CGFloat const RatingsViewHeight                  = 100.0f;
+
+typedef NS_ENUM(NSUInteger, NotificationFilter)
+{
+    NotificationFilterNone      = 0,
+    NotificationFilterUnread    = 1,
+    NotificationFilterComment   = 2,
+    NotificationFilterFollow    = 3,
+    NotificationFilterLike      = 4
```

Comment:
Any reason for the assignments? They would be the values of the enum even if not explicitly set, yah?